README.md

```diff
@@ -1 +1,54 @@
-# rudder-ingester
+# rudder-load
+
+This repository is used to generate an artificial load to test the RudderStack architecture under intense traffic.
+
+## Usage
+
+The service is designed to be run in a Kubernetes cluster.
+
+It is possible to create multiple replicas of the service to increase the load.
+
+Each replica owns a specific writeKey and will send events via multiple go routines. Each go routine will send events
+owning a set of keys. This way we can guarantee in-order delivery of events per writeKey-userId.
+
+## Example configuration
+
+1. Replicas: 3
+2. WriteKeys: 3
+3. Concurrency: 200 (number of go routines per replica, we'll call them **slots**)
+4. Message generators: 10
+   a. number of go routines that will generate events and send them to the **slots** (see point 3)
+5. Keys per slot map: 50
+   a. number of **unique** keys that each slot will own
+   b. this can be a map as well like 10_20_30_40 which would mean that 50 slots will own 10 keys, 
+      50 slots will own 20 keys, 50 slots 30 keys and 50 slots 40 keys (because we create 4 groups,
+      and we have 200 slots)
+6. Traffic distribution percentage: 60_40
```

Comment:
how can we define the distribution for diffrent event types ?

Reply:
You can't. Now you can either send a `page` or a `batch` with inside the same `page` over and over (only `messageId` will change).